aggsender/metrics/metrics.go

```diff
@@ -77,3 +120,35 @@ func CertificateBuildTime(value float64) {
 func ProverTime(value float64) {
 	prometheus.GaugeSet(proverTime, value)
 }
+
+// CertificateSettlementTime sets the gauge for the certificate settlement time
+func CertificateSettlementTime(value float64) {
+	prometheus.GaugeSet(certificateSettlementTime, value)
+}
```

Comment:
Same as the certificates validation, it is worth considering using the histogram. WDYT?